cli/options.go

```diff
@@ -480,6 +491,13 @@ func withConvertWindowsPaths(options *ProjectOptions) func(*loader.Options) {
 	}
 }
 
+// save listeners from ProjectOptions (compose) to loader.Options
+func withListener(options *ProjectOptions) func(*loader.Options) {
```

Comment:
As there could be more than one listener to save
```suggestion
func withListeners(options *ProjectOptions) func(*loader.Options) {
```

Reply:
indeed. I plan to add `include` metric support using this mechanism so will fix this at same time in a follow-up PR